datasets/br/bcb_disqualified_persons/bcb_disqualified_persons.yaml

```
@@ -0,0 +1,27 @@
+title: Brazil's Central Bank General Register of Disqualified Persons  
+entry_point: crawler.py
+prefix: bcb-disqualified-persons
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  The QGI is a dataset from the Central Bank of Brazil listing individuals disqualified from certain roles in financial institutions and consortium administrators.
+description: |
+  The QGI, maintained by the Central Bank of Brazil,
+  serves as a crucial resource for monitoring regulatory compliance in the financial sector.
+  It enumerates individuals who, due to penalties incurred in administrative sanction proceedings,
+  are temporarily prohibited from occupying key positions such as administrators
+  or board members in financial or consortium administration entities.
+  This measure ensures the integrity and reliability of personnel in critical financial roles,
+  reflecting the Central Bank's commitment to maintaining stability and transparency in the financial system.
+  The dataset is a part of broader efforts to uphold high standards in financial governance
+  and protect the interests of stakeholders in the financial industry.
+publisher:
+  name: BCB (Brazil's Central Bank)
+  url: https://www.bcb.gov.br
+  official: true
+  country: "br"
+url: https://olinda.bcb.gov.br/olinda/servico/Gepad_QuadrosGeraisInternet/versao/v1/odata/QuadroGeralInabilitados?format=text/html
+data:
+  url: https://olinda.bcb.gov.br/olinda/servico/Gepad_QuadrosGeraisInternet/versao/v1/odata/QuadroGeralInabilitados?format=text/html
```

I suspect it would work just as well without this, because it's probably using the accept header and ignoring the querystring. Then this querystring is a bit confusing given the code's using JSON.

```suggestion
  url: https://olinda.bcb.gov.br/olinda/servico/Gepad_QuadrosGeraisInternet/versao/v1/odata/QuadroGeralInabilitados
```